apps/wpcom-block-editor/README.md

```diff
@@ -4,36 +4,37 @@ This package provides utilities for the WordPress.com block editor integration.
 
 These utilities are intended to be built and then served from `widgets.wp.com`, so they can be loaded by a WordPress.com or a Jetpack connected site.
 
-## File architecture
+## Environments
+
+There are two environments the block editor integration supports:
 
-### `iframe-bridge-server.js`
+- **WP Admin block editor**. The block editor loaded by the WP Admin interface on `https://<SITE_SLUG>/wp-admin/post-new.php`.
+- **Calypso block editor**. This is the block editor loaded by Calypso on `https://wordpress.com/block-editor/post/<SITE_SLUG>`. It is actually the WP Admin block editor embed on an iframe. We also refer to this implementation as _Gutenframe_.
+ 
+## File architecture
 
-Server-side handlers of the different communication channels we establish with the client-side when Calypso loads the iframed block editor. See [`calypsoify-iframe.jsx`](https://github.com/Automattic/wp-calypso/blob/master/client/gutenberg/editor/calypsoify-iframe.jsx).
+- `/common`: Logic than runs on both environments (WP Admin and Calypso).
+- `/calypso`: Logic than runs only on the Calypso iframed block editor. 
+- `/wp-admin`: Logic than runs only on the WP Admin block editor. 
```

Comment:
Might as well remove this, since it's not present, and not anticipated in the near future.